### DISCUSSION TITLE
A mod who change the song's pitch but not the speed & 'proportional pitch' option for DC and NC mods according to the speed.

### DISCUSSION THREAD

**Discussion by Corne2Plum3** [ppy/osu#13142] [Ideas] [answered]
It would be nice if there's a mod with a 1.00x multiplier (doesn't affects the score) where we can set the pitch of the song (between 0.5 and 2.0) without affecting the song's speed, like nightcore or wind up for exemple.

I have a 2nd idea, for Nightcore and Daycore mods. It's an option 'proportional pitch', where the pitch of the song is modified according to the selected speed instead of a fixed pitch (1.5x for NC and 0.75x for DC), as they does for now.

_(2022/05/29 edit: improved grammar and clarity of the suggestion, and improved title)_

**Comment by Walavouchey** [comment, marked as answer]
both already suggested in https://github.com/ppy/osu/discussions/17883 and https://github.com/ppy/osu/discussions/13125 respectively
